compositions/cluster-k8s/big-blocks/tc-002-da-sync/100-100-50-1000/320-100mib-200ms.toml

```diff
@@ -14,12 +14,34 @@
   execution-time = "10"
   persistent-peers = "10"
   submit-times = "12"
-  msg-size = "70000"
+  msg-size = "64000"
+  seed = "5"
   validator = "100"
   bridge = "100"
   full = "50"
   light = "1000"
 
+[[groups]]
+id = "seeds"
+builder = "docker:generic"
+[groups.resources]
+memory = "4Gi"
+cpu = "3"
+[groups.instances]
+count = 5
+percentage = 0.0
+[groups.build_config]
+build_base_image = "golang:1.19.1"
+enable_go_build_cache = true
+enabled = true
+go_version = "1.19"
+[groups.build]
+[groups.run]
+[groups.run.test_params]
+latency = "200"
+bandwidth = "320Mib"
+role = "seed"
+
```

Comment:
Blocking: It feels like a NIT, but I have to insist 😢

Reply:
done